pkg/encryption/encryption.go

```diff
@@ -518,24 +518,29 @@ func (s *Service) areAllMasterNodesUsingLatestConfig(ctx context.Context, wcClie
 		return false, microerror.Mask(err)
 	}
 
-	// get all master nodes
-	var nodes v1.NodeList
-	err = wcClient.List(ctx,
-		&nodes,
-		ctrlclient.MatchingLabels{key.MasterNodeLabel: ""})
-	if err != nil {
-		return false, microerror.Mask(err)
+	// get all master nodeItems
+	nodeItems := []v1.Node{}
+	for _, label := range key.MasterNodeLabels {
+		var tmpNodes v1.NodeList
+		err = wcClient.List(ctx,
+			&tmpNodes,
+			ctrlclient.MatchingLabels{label: ""},
```

Comment:
I guess the `""` should be `label` instead?

Reply:
I don't think so.

`MatchingLabels` is a `map[string]string`, which maps label keys to label values.
The controlplane nodes have the key-value-pair `node-role.kubernetes.io/control-plane: ""` in  `metadata.labels[]`.

Or am I missing something here?

Note, that before my change it also was:
```
		ctrlclient.MatchingLabels{key.MasterNodeLabel: ""})
```

Reply:
yeah right, dunno what I saw